src/import-data/useImportRequest.ts

```
@@ -111,10 +109,6 @@ const getTDRSnapshotExportImportRequest = async (queryParams: QueryParams): Prom
     throw new Error('Unable to load snapshot.');
   }
 
-  if (snapshot.cloudPlatform === 'azure' && !isFeaturePreviewEnabled(ENABLE_AZURE_TDR_IMPORT)) {
```

Do we need to throw an error in this method too like we do below?

---

Added: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/c1edb7478152efd0d5b758e503c89fecb8831688